### ISSUE TITLE
What's the policy on claiming trademarked or business names?

### ISSUE THREAD

**Issue by tgvashworth** [appdotnet/api-spec#72]
Kinda self-explanatory.

For example, could I have registered as @google on ADN?

**Comment by abitgone** [comment]
Ha, I beat you to it for usernames @phuu – see issue #71 – though your question about trademarks/business names is most valid and very interesting.

**Comment by tgvashworth** [comment]
@abitgone Hah! I'll switch mine over to solely about business names.

**Comment by wpstudio** [comment]
Or let's take it a little deeper. What if someone registers @news on ADN but someone else has @news on Twitter and now they want @news on ADN... or what if a building contractor who installs windows takes the @windows handle on ADN BEFORE Microsoft picks up @windows on ADN?  How are disputes handled? Or even the issue of regular usernames, what if a 15 year old developer from Ohio named Bill Gates registers @billgates - does that mean the other Bill Gates is out of luck?

**Comment by chrishannah** [comment]
Surely it should be just a simple first come, first serve basis. Why should Microsoft get priority over @windows for example?

**Comment by tgvashworth** [comment]
@cryptixdevelopment Trademarks for one. Apparently you can grab a domain if you own the TM, and a Twitter handle, so why not an ADN handle? I'm just interested in the policy.

**Comment by ghost** [comment]
I think there would be a lot of lessons to take from domains. You could get @windows but if you were using it against trademark law then... 

Also maybe for now it's simple pay to stay, 30-n days to reactivate or the username is let go.

**Comment by chrishannah** [comment]
@phuu I never knew that about twitter handles, I wonder if TM law will overpower the usage of certain usernames or if it will be down solely to ADNs policies.

**Comment by abitgone** [comment]
I'd say this has a lot more to do with trademark law in a given jurisdiction, but then you get into the issues of having multiple jurisdictions (potentially) fighting over the same trademark. It's easy to say "first come, first served" but trademark law is full of very complex legal issues, and this is an area which will probably only become clearer and more transparent once the first dispute arises. 

That said, having a dispute resolution process outlined ahead of time would go a long way towards solving the issue.

**Comment by conradkleinespel** [comment]
@cryptixdevelopment @phuu As long as the ADN handle @windows does not try to make people think it is part of Microsoft, why would Microsoft get the name? They could just get MicrosoftWindows. Especially since anyone that likes or sells regular windows for houses should be able to claim that handle too, at least IMO.

**Comment by chrishannah** [comment]
@conradkleinespel I agree. I think if it doesn't directly relate to the trademark then it should be first come first served basis, but if someone took @MicrosoftWindows then there might be an issue. Just the same as anyone should be able to get @apple but maybe not @AppleInc.

**Comment by jschlesser** [comment]
Have fun with that, sounds like an invite to spend a bunch of time lawyers to me.  Windows is most likely trademarked as well as apple in a variety of contexts.
